src/Cardano/Wallet/CoinSelection/Fee.hs

```diff
@@ -421,10 +419,7 @@ estimateFee policy (CoinSelection inps outs chngs) =
     --  | word64                      -- 1|2|3|5|9
     sizeOfTxOut :: TxOut -> Int
     sizeOfTxOut (TxOut (Address bytes) c) =
-        6
-        + BS.length bytes
-        + sizeOf (CBOR.encodeWord32 $ crc32 bytes)
-        + sizeOfCoin c
+        1 + BS.length bytes + sizeOfCoin c
```

Comment:
So, turns out those 5 bytes + the crc are already part of the `bytes` payload! So we don't need to count them twice!